web/portal/client/src/entities/HolidayDate/Action/AddHolidayDateRange.tsx

```diff
@@ -31,24 +33,26 @@ const AddHolidayDateRange: ActionFunctionComponent = (
 
   return (
     <>
-      <Link to={targetUrl} style={linkStyles}>
-        {variant === 'text' && (
-          <MoreMenuItem>{_('Add Holiday date range')}</MoreMenuItem>
-        )}
-        {variant === 'icon' && (
-          <Tooltip
-            title={_('Add Holiday date range')}
-            placement='bottom'
-            enterTouchDelay={0}
-          >
-            <span>
-              <StyledTableRowCustomCta>
-                <AddCircleIcon />
-              </StyledTableRowCustomCta>
-            </span>
-          </Tooltip>
-        )}
-      </Link>
+      {aboutMe?.restricted && (
```

Comment:
Not restricted users will always see this option enabled.
Restricted admins must have the proper aboutMe.acls in order to see it.
Show it disabled otherwise.